src/GameDataReader/Common/Refractor/V1/Files/UtilityMethodsRefractorV1ConfigFile.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace GameDataReader.Common.Refractor.V1.Files;
+
+internal class UtilityMethodsRefractorV1ConfigFile
+{
+    public string GetProcessPath(string modName)
```

Comment:
While the attribute of the config file is called `_modName`, I don't think it makes sense to use that as parameter name here. The method is totally generic, so it should be something like `processName`.